src/net.h

```diff
@@ -113,6 +113,14 @@ struct CSerializedNetMsg
     std::string command;
 };
 
+enum class ConnectionType {
+    INBOUND, // peer initiated connections
+    OUTBOUND, // full relay connections (blocks, addrs, txns)
+    MANUAL, // connections to addresses added via addnode or the connect command line argument
+    FEELER, // short lived connections used to test address validity
+    BLOCK_RELAY, // only relay blocks to these connections
```

Comment:
Add that these are automatic outbound peers made to addresses selected from AddrMan.

Reply:
added